tests/sesolve/closed_system.py

```diff
@@ -0,0 +1,64 @@
+from abc import ABC, abstractmethod
+
+import numpy as np
+import qutip as qt
+import torch
+from torch import Tensor
+
+import torchqdynamics as tq
+
+
+class ClosedSystem(ABC):
+    def __init__(self):
```

Comment:
Right, I think we may want to promote the use of `torch.set_default_dtype` rather than passing dtypes everywhere, we can discuss this next meeting, I'll add it to the agenda. For tests it might be better to have a `dtype` argument, I chose not to care for now as we care nowhere in the code, but we definitely need to fix this at some point.

Reply:
TL;DR I think it needs a global discussion rather than just a fix here, so I propose to postpone 😉

Reply:
Alright let's organize a meeting next week if you guys are available. There's quite a few things to discuss by now I think.